modules/ezcontent_node/modules/ezcontent_smart_article/src/Form/ConfigForm.php

```diff
@@ -220,6 +220,16 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#options' => $imageTaggingOptions,
       '#default_value' => $config->get('image_tagging_service'),
     ];
+    $form['image_tagging_action_type'] = [
+      '#type' => 'radios',
+      '#title' => $this->t('Image Tagging Action Type'),
+      '#description' => $this->t('Select an action type.'),
+      '#options' => [
+        'auto' => $this->t('Generate image tags as soon as image is uploaded.'),
+        'button_click' => $this->t('Generate image tags on button click after image is uploaded.'),
```

Comment:
Can we update the text to `Generate image tags, manually, by clicking a button.`